src/main/java/com/bhoruka/bloodbank/model/response/SearchCampResponse.java

```diff
@@ -0,0 +1,26 @@
+package com.bhoruka.bloodbank.model.response;
+
+import java.util.Date;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class SearchCampResponse {
+
+    String id;
+
+    String partnerId;
+
+    Date dateOfCamp;
+
+    Long expectedNoOfDonors;
+
+    Long actualNoOfDonors;
+
+    Integer pageNo;
+
+    Integer pageSize;
+
+    String sortBy;
```

Comment:
Why is this required in the response?

Reply:
Shouldn't we show on which basis the page is sorted?

Reply:
the client will be passing the sortBy field to you. How does it add any value to return the same thing back in the response?

Reply:
Point